hack/csi_client/cmd/main.go

```diff
@@ -14,7 +15,7 @@ Usage of %s:
 	Other necessary flags are determined based on the request type.
 
 	NodePublish
-		- shareAddr, shareName, targetPath, smbUsername (WINDOWS), smbPassword (WINDOWS)
+		- volumeAttr, targetPath, secrets (Windows)
```

Comment:
Can you update the README with an example for Filestore?

Reply:
Done.